helpers_test.go

```diff
@@ -0,0 +1,388 @@
+package validate
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func stringPtr(s string) *string {
+	return &s
+}
+
+func TestRequireField(t *testing.T) {
```

Comment:
This test suite is really nice, makes the behaviour of the function really clear (because its not that clear from the code!)

Reply:
Sounds like the method needs a bit better godoc comment :)